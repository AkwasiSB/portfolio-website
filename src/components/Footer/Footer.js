import React from 'react';
import styles from './Footer.scss';

export default class Footer extends React.Component {
	render() {
		return (
			<footer className={styles.footer}>
				<p className={styles.footerText}>
					Â© 2019 Akwasi Sefa-Boakye. All rights reserved.
				</p>
			</footer>
		);
	}
}
