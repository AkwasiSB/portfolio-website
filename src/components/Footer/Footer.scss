.footer {
	grid-area: footer;
	background-color: #000;
	display: flex;
	justify-content: center;
	align-items: center;

	&-text {
		color: #fff;
		font-size: 1.3rem;
	}
}
