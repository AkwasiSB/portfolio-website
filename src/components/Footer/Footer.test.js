import Footer from './Footer';

it('should render <Footer /> correctly', () => {
  const wrapper = shallow(<Footer />);

  expect(wrapper).toMatchSnapshot();
});
