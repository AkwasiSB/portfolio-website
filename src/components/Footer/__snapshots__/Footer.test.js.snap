// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <Footer /> correctly 1`] = `
<footer
  className="footer"
>
  <p
    className="footerText"
  >
    Â© 2019 Akwasi Sefa-Boakye. All rights reserved.
  </p>
</footer>
`;
